tests/e2e/e2e_ibc_test.go

```diff
@@ -276,6 +293,192 @@ func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 	})
 }
 
+/*
+TestMultihopIBCTokenTransfer tests that sending an IBC transfer using the IBC Packet Forward Middleware accepts a port, channel and account address
+
+Steps:
+1. Check balance of Account 1 on Chain 1
+2. Check balance of Account 2 on Chain 1
+3. Account 1 on Chain 1 sends x tokens to Account 2 on Chain 1 via Account 1 on Chain 2
+4. Check Balance of Account 1 on Chain 1, confirm it is original minus x tokens
+5. Check Balance of Account 2 on Chain 1, confirm it is original plus x tokens
+
+*/
+func (s *IntegrationTestSuite) TestMultihopIBCTokenTransfer() {
+	time.Sleep(30 * time.Second)
+
+	// var ibcStakeDenom string
```

Comment:
Looks like a bunch of commented out variables...not sure if these need to be commented in later for something or they're left over

Reply:
good catch, used them in previous versions of the test, removed now.